tests/runtime/CMakeLists.txt

```diff
@@ -233,6 +233,7 @@ if(FLB_IN_LIB)
     # These plugins work only on Linux
   if(NOT FLB_SYSTEM_WINDOWS)
     FLB_RT_TEST(FLB_OUT_FILE              "out_file.c")
+    FLB_RT_TEST(FLB_OUT_LOGROTATE         "out_logrotate.c")
   endif()
```

Comment:
I think this is outdated right as we seem to enable them on windows @cosmo0920 ?